### PR TITLE
NavBar Version 1

### PR DESCRIPTION
**Pull request by harri012**
- Created an interactive Navbar using React bootstrap
- Linked the Navbar interface with their respective links 
- Created temporary .js files for missing linkers if needed
- Updated a readme typo

To update later:
- Add Heagle logo
- add icons for cart, user
- edit styling (fonts, colour and more)

Screenshots:
image
image

### CODE REVIEW

**Comment by harri012** [comment]
Also downloaded bootstrap folders to project

File: ecommerce-website/public/index.html [hunk 1]
```diff
@@ -24,6 +24,18 @@
...
+    <script
+      src="https://unpkg.com/react-dom/umd/react-dom.production.min.js"
+      crossorigin></script>
+
+    <script
+      src="https://unpkg.com/react-bootstrap@next/dist/react-bootstrap.min.js"
+      crossorigin></script>
+
+<script>var Alert = ReactBootstrap.Alert;</script>
+
```
Comment (Its-Alann) [inline ecommerce-website/public/index.html:38]: What are these files, do you need them?

**Review by Its-Alann — CHANGES_REQUESTED** [review]
Very nice header, great job. Just have a question about some lines in the index,html. Let's wait for #124 to be merged first since it has some common files and we can merge it after resolving the conflicts.

File: ecommerce-website/public/index.html (hunk 1 above)
Comment (harri012) [reply ecommerce-website/public/index.html]: Removed them. It was a double and didn't see it was included in modules already.

**Review by Its-Alann — APPROVED** [review]
Thanks for the changes

**Review by WillTrem — APPROVED** [review]
Awesome! Looks very nice! Great work

**Pull request merged**